web/controllers/api/owner_controller.ex

```diff
@@ -27,8 +27,10 @@ defmodule HexWeb.API.OwnerController do
   def create(conn, %{"email" => email}) do
     email = URI.decode_www_form(email)
     user = HexWeb.Repo.get_by!(User, email: email)
+    package = conn.assigns.package
 
-    Package.create_owner(conn.assigns.package, user) |> HexWeb.Repo.insert!
+    Package.create_owner(package, user) |> HexWeb.Repo.insert!
+    HexWeb.AuditLog.create!(conn, "owner.add", {package, user})
```

Comment:
@ericmj regarding your 2 comments about not accessing Repo in Models, and passing in user explicitly. I think the usage in controllers would then be something along the lines of:

```elixir
HexWeb.Repo.transaction(fn ->
  Package.create_owner(package, user) |> HexWeb.Repo.insert!
  HexWeb.AuditLog.build(conn.assigns.user, "owner.add", {package, user}) |> HexWeb.Repo.insert!
end)
```

The audit invocation is pretty verbose and its gonna be in almost every controller, so I'm thinking about potentially moving some boilerplate to a helper, but will have a better idea if it's worth it when I update all controllers first with transactions